main.cpp

```diff
@@ -10,158 +9,236 @@ video: Chapter 2 - Part 3
-    float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar ); return { }; }
-    This does not conform with the coding standard for this course (check the Readme.MD) and needs to be corrected
-    
-9) in the main function at the end: 
-    for each of those functions declared, 
-        a) write out how the function would look if called with correct arguments
-        b) if the function returned anything, store it in a local variable via the 'auto' keyword.
-        c) pass the local variables to ignoreUnused() as you did in 2b)
-    see main() for an example of this.
- 
+4) add { ignoreUnused( ); } after each declaration in place of the closing
+semicolon 5) pass each of your function parameters to the ignoreUnused function
+like you did in b) 6) if your function returns something other than void, add
+'return { };' at the end of it. 7) provide default values for an arbitrary
+number of parameters in the function parameter list.
+
+8) consult the coding style guide found in the Readme.MD and adjust the
+formatting of your 10 functions.  At this point, you might have something that
+looks like this: float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar );
+return { }; } This does not conform with the coding standard for this course
+(check the Readme.MD) and needs to be corrected
+
+9) in the main function at the end:
+    for each of those functions declared,
+        a) write out how the function would look if called with correct
+arguments b) if the function returned anything, store it in a local variable via
+the 'auto' keyword. c) pass the local variables to ignoreUnused() as you did in
+2b) see main() for an example of this.
+
 10) click the [run] button.  Clear up any errors or warnings as best you can.
  */
 
-//2)
-void variableDeclarations()
-{
-    //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+// 2)
+void variableDeclarations() {
+  // example:
+  int number =
+      2; // declaration of a variable named "number", that uses the primitive
+         // type 'int', and the variable's initial value is '2'
+  int newNumber = 4;
+  int largeNumber = 200;
+  int smallNumber = 1;
+  float floatNumber = 2.2f;
+  float largeFloatNumber = 200.2f;
+  float smallFloatNumber = 1.1f;
+  char goodBye = 'g';
+  char hello = 'h';
+  char welcome = 'w';
+  double doubleNumber = 2.222;
+  double largeDoubleNumber = 200.2;
+  double smallDoubleNumber = 1.1;
+  bool truth = true;
+  bool lie = false;
+  bool maybe = true;
+
+  ignoreUnused(
+      newNumber, largeNumber, smallNumber, floatNumber, largeFloatNumber,
+      smallFloatNumber, goodBye, hello, welcome, doubleNumber,
+      largeDoubleNumber, smallDoubleNumber, truth, lie,
+      maybe); // passing each variable declared to the ignoreUnused() function

```

Comment:
Please review the course coding standard found in the README.MD file.
It looks like you were using an auto formatter. 
Please adjust the settings so that it matches the course coding standard. Code that does not conform to the coding standard will not be approved. 
- [ ] revise